.github/workflows/release.yaml

```diff
@@ -75,68 +70,47 @@ jobs:
             runner: warp-ubuntu-latest-x64-16x
           - target: aarch64-unknown-linux-gnu
             runner: warp-ubuntu-latest-arm64-16x
-          - target: aarch64-apple-darwin
-            runner: warp-macos-14-arm64-6x
+          # Paused until docker is pre-installed https://github.com/actions/runner-images/blob/main/images/macos/macos-14-arm64-Readme.md
+          # - target: aarch64-apple-darwin
+          #   runner: warp-macos-14-arm64-6x
         features:
           - ""
-          - "redact_sensitive"
+          - "redact-sensitive"
 
     steps:
-      - name: Checkout sources
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0 # needed for built.rs to get GIT_HEAD_REF
-
-      # https://github.com/dtolnay/rust-toolchain
-      - name: Setup rust toolchain
-        uses: dtolnay/rust-toolchain@stable
-        with:
-          target: ${{ matrix.configs.target }}
-
-      # https://github.com/WarpBuilds/rust-cache
-      - name: Run WarpBuilds/rust-cache
```

Comment:
Why is this removed?

Reply:
good question. it's not significantly different. we're building in parallel for [5-10 minutes](https://github.com/flashbots/rbuilder/actions/runs/12130028687) per binary, which is a ~1 minute longer. we can reintroduce with a bit of work if build times become an issue. 
![image](https://github.com/user-attachments/assets/c63801ae-9bcb-421d-8372-0f6a8f99345e)